known-shortcuts/obsidian.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:zap: **Suggestion** 
The `app` object is being cast to `AppParams` using `as`. It would be more type-safe to define the `app` object with the `AppParams` type directly. This ensures that all properties conform to the `AppParams` type at the time of object creation.